tests/test_scripts.py

```diff
@@ -12,7 +12,7 @@
 SCRIPTS = [
     'creategoogleauthenticator-file',
     'getgooglecodes',
-    'privacyidea-convert-base32.py',
+#    'privacyidea-convert-base32.py',
```

Comment:
Why is that? Should we or do we need to remove this? Or did we remove this?

Reply:
This is what breaks the current tests. This script requests some input from stdin which confuses the test-runner. So i just commented it out for now.